README.md

```diff
@@ -45,7 +62,7 @@ the field.
 *********************************************/
 
 var tessel = require('tessel');
-var pir = require('../').use(tessel.port['GPIO'].pin['G3']);
+var pir = require('../').use(tessel.port['A'].pin[1]);
```

Comment:
it doesn't matter; any [digital pin](https://tessel.io/docs/hardwareAPI#pin-mapping) will work. Is there somewhere this should be made more clear?